resources/tenant-onboarding.yaml

```diff
@@ -18,13 +18,16 @@ Parameters:
   Environment:
     Description: Environment (test, uat, prod, etc.)
     Type: String
+    Default: oct12
```

Comment:
These changes were removed by https://github.com/awslabs/aws-saas-boost/commit/d0d0478fd1d20e5c07d9409974cc3ed1d4e0b21b. Is your fork up to date with main?